modules/structs/repo_branch.go

```diff
@@ -125,3 +128,8 @@ type EditBranchProtectionOption struct {
 	UnprotectedFilePatterns       *string  `json:"unprotected_file_patterns"`
 	BlockAdminMergeOverride       *bool    `json:"block_admin_merge_override"`
 }
+
+// UpdateBranchProtectionPriories a list to update the branch protection rule priorities
+type UpdateBranchProtectionPriories []struct {
```

Comment:
Again, here we need a proper struct, instead of the naked array.

Reply:
@wxiaoguang is https://github.com/go-gitea/gitea/pull/32286/commits/c1bda05feeddb8f5944ce2f93fb7b264c97506c7 what you ment?